packages/server/api/src/app/workers/worker-controller.ts

```diff
@@ -102,6 +102,7 @@ export const flowWorkerController: FastifyPluginAsyncTypebox = async (app) => {
         )
         const createFlowRuns = filterPayloads.map((payload) =>
             flowRunService(request.log).start({
+                flowRunId: apId(),
```

Comment:
Why you specify this? I think the method has logic if this is preexisting one for some reason